etc/downloads_badges.py

```diff
@@ -3,6 +3,7 @@
 import math
 import os
 import requests
+from statistics import mean
```

Comment:
Lol I didn't even know this was a builtin

Reply:
What would you have used, numpy or something?